### PR TITLE
Planning: removed repeated obstacle vectors in open space ROI.

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/planning/common/open_space_info.h [hunk 1]
```diff
@@ -292,7 +292,17 @@ class OpenSpaceInfo {
     return &debug_instance_;
   }
 
-  void sync_debug_instance() { debug_instance_.MergeFrom(*debug_); }
+  void sync_debug_instance() {
+    // Remove existing obstacle vectors to prevent repeating obstacle
+    // vectors.
+    if (debug_->planning_data().open_space().obstacles().size()) {
```
Comment (xiaoxq) [inline modules/planning/common/open_space_info.h]: Use `!debug_->planning_data().open_space().obstacles().empty()`.

File: modules/planning/common/open_space_info.h [hunk 2]
```diff
@@ -292,7 +292,17 @@ class OpenSpaceInfo {
...
+      debug_instance_.mutable_planning_data()
+          ->mutable_open_space()
+          ->mutable_obstacles()
+          ->Clear();
```
Comment (xiaoxq) [inline modules/planning/common/open_space_info.h]: IIRC, `debug_instance_.mutable_planning_data()->mutable_open_space()->clear_obstacles()` is better.

Comment (sjiang2018) [reply modules/planning/common/open_space_info.h]: Fixed. Thank you.

File: modules/planning/common/open_space_info.h (hunk 1 above)
Comment (sjiang2018) [reply modules/planning/common/open_space_info.h]: Fixed. Thank you.

**Pull request merged**